aries-site/src/pages/foundation/background-colors-guidance.mdx

```diff
@@ -65,14 +93,31 @@ above a main content section that uses `background-back`.
 Refer to the [background palette](/foundation/color#background-palette) for definition
 of each background color namespace and value.
 
-### Example of layered layout
+## Examples of Layered Layout
 
 For example, a page with a [header](/components/header), summary content, and a grid of
 [cards](/components/card) would use `background-front` for each card and
 `background-back` for everything else.
 
 For a multi-column page layout, `background-front` could be used to distinguish the column which should be the primary focus.
 
+### With fixed header
+
+When the header is fixed, use `background-front` for the header since it is layered in front of the main content.
```

Comment:
Do we want a color swatch like you used in the example description above?

```suggestion
When the header is fixed, use `background-front` <ColorSwatch background="background-front" /> for the header since it is layered in front of the main content.
```